src/DIRAC/Resources/Storage/GFAL2_StorageBase.py

```diff
@@ -50,6 +51,54 @@
     ECOMM = 70
 
 
+@contextmanager
+def setGfalSettings(ctx, pluginName, optionName, optionValue):
```

Comment:
Would it be common to set multiple settings at the same time? If so would this signature make more sense?

```python
def setGfalSettings(ctx, pluginName, **options):
    for optionName, optionValue in options.items():
        ...
```